tests.md

```diff
@@ -0,0 +1,833 @@
+         it 'Delegating script address - any' do
+         it 'Enterprise pub key address' do
+         it 'Reward account pub key address' do
+         it 'Delegating address with both pub key credentials' do
+         it 'Delegating address - payment from script, stake from key' do
+         it 'Delegating address - payment from key, stake from script' d
+       it 'Malformed payload when tx is not binary' do
+```
+
+
+```
+shared:      it 'I can create, get and delete wallet from mnemonics getting acc_xpub from cardano-address' do
+         it 'I can create, get and delete wallet from pub key getting acc_xpub from cardano-address' do
+         it 'Cannot create wallet with different acc xpub - derived from different mnemonic sentence' do
+         it 'Cannot create wallet with different acc xpub - derived from different acc ix' do
+         it 'I can create incomplete wallet and update cosigners with acc_xpub from cardano-address' do
+         it 'Create / update partially / get / list / delete' do
+         it 'Cannot update main cosigner' do
+           it 'Shared walletid with only spending template from cardano-addresses' do
+           it 'Shared walletid with spending and delegation template from cardano-addresses' do
+         it 'Can see utxo' do
+         it 'Can see utxo snapshot' do
+         it 'Can list addresses on active shared wallet - from pub key' do
+         it 'Can list addresses on active shared wallet - from mnemonics' do
+         it 'Lists empty addresses on incomplete shared wallet - from pub key' do
+         it 'Lists empty addresses on incomplete shared wallet - from mnemonics' do
+         it 'Get public key - incomplete wallet from mnemonics' do
+         it 'Get public key - incomplete wallet from acc pub key' do
+         it 'Get public key - active wallet from mnemonics' do
+         it 'Get public key - active wallet from acc pub key' do
+         it 'Create account public key - incomplete wallet from mnemonics' do
+         it 'Cannot create account public key - incomplete wallet from acc pub key' do
+         it 'Create account public key - active wallet from mnemonics' do
+         it 'Cannot create account public key - active wallet from acc pub key' do
+         it 'Get account public key - active wallet from mnemonics' do
+         it 'Get account public key - active wallet from acc pub key' do
+         it 'Get account public key - incomplete wallet from mnemonics' do
+         it 'Get account public key - incomplete wallet from acc pub key' do
+       it 'I could get a tx if I had proper id' do
+       it 'Can list transactions' do
+e2e:    it 'ADP-2523 - Make sure there are no null values in the response', :adp_2523 do
+        it 'AlwaysFails.plutus with collateral return to the wallet' do
+        it 'cannot balance on empty wallet' do
+        it 'ping-pong' do
+        it 'game' do
+        it 'mint-burn' do
+        it 'withdrawal' do
+        it 'currency' do
```

Comment:
These are currently no longer tested with integration tests: https://github.com/cardano-foundation/cardano-wallet/blob/master/lib/integration/scenarios/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs#L2987-L3045 (`withdrawal` not there, the others exist but are marked pending)